### ISSUE TITLE
Export to Excel

### ISSUE THREAD

**Issue by RoMoAmy** [TablePlus/TablePlus#544]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): MySQL 5.7

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): 115

Are there plans to allow the export of data to Excel?

**Comment by chrschmitz** [comment]
I would like to bump this feature. And please consider also an Excel import, preferably for multiple Excel files at the same time (see Navicat).

**Comment by johnakitto** [comment]
Definitely would _love_ a .XLSX export option... That would be amazing.

**Comment by huyphams** [comment]
Let me see if I can make it on the next release, thanks guys

**Comment by jrz** [comment]
Yeah, +1 for me.. Would be great to have an excel export where every selected table is a sheet. Preferrably with quickfilters already set up, and the proper datatype for each cell/column.

**Comment by darryldale** [comment]
Agreed 100%

An export to excel feature (for my workflow) is a required feature.

**Comment by darryldale** [comment]
> Let me see if I can make it on the next release, thanks guys

I see this was something you were considering back in April. Is this still a feature you are planning on adding?

**Comment by darryldale** [comment]
Any update?

**Comment by zhangsean** [comment]
Looking forward to this feature.

**Comment by migrant** [comment]
Looking forward

**Comment by MrGreyKnight** [comment]
+1 Would be a greatly useful feature, only thing missing for us at the moment

**Comment by gn0rt0n** [comment]
Would live the ability to import/export from excel. This is a big reason I still use Navicat as part of the workflow.

**Comment by jordanhavard** [comment]
+1 Would be great

**Comment by 3riik** [comment]
You can do export to CSV and use semicolon ; as delimiter

**Comment by DanielMendezLV** [comment]
An export to excel feature would be amazing.

**Comment by jonasspott** [comment]
Would love this feature!

**Comment by nousename** [comment]
Would also appreciate this

**Comment by GenaroHV** [comment]
@huyphams alguna novedad?

**Comment by ahmad-mujamil** [comment]
any update on this ?

**Comment by huyphams** [comment]
We've implemented it, and it will be available in the next update!
